HW2/P5/driver.py

```diff
@@ -77,6 +110,8 @@ def randcolor():
             print("{} simulation frames per second".format(frame_count / total_time))
             frame_count = 0
             total_time = 0
-            # SUBPROBLEM 3: sort objects by location.  Be sure to update the
-            # grid if objects' indices change!  Also be sure to sort the
-            # velocities with their object positions!
+
```

Comment:
Is there a reason you didn't put the code in the "subproblem 3 section?" Maybe it's better to sort every update and not just on animation steps.